README.md

```diff
@@ -15,10 +16,44 @@ Then alter your homeserver configuration, adding to your `modules` configuration
 modules:
   - module: synapse_user_restrictions.UserRestrictionsModule
     config:
-      # Optional: example boolean flag
-      example_option: True
+      # List of rules. Earlier rules have a higher priority than later rules.
+      rules:
+        - match: "@admin.*:example.org"
+          allow: [invite, create_room]
+
+        - match: "@assistant.*:example.org"
+          allow: [invite]
+
+      # If no rules match, then these permissions are allowed.
+      default_allow: []
+
+      # If no rules match, then these permissions are denied.
+      default_deny: [invite, create_room]
 ```
 
+In this example:
+- `@adminalice:example.org` could create rooms and invite users to
+rooms;
+- `@assistantbob:example.org` could invite users to rooms but NOT create rooms;
+and
+- `@plainoldjoe:example.org` could neither create rooms nor invite users.
+
+### Configuration
+
+Rules are applied top-to-bottom, with the first matching rule being used.
+
+A rule matches if the regular expression (written in `match`) matches the user's ID,
+and the permission being sought is either in the `allow` list or the `deny` list.
```

Comment:
Yeah, it is. I had to look up the documentation to figure out which one is which in Python's `re` library, further illustrating how good it would be to document :)